src/cluster/serializer/broker/Serializer.cc

```diff
@@ -96,7 +95,7 @@ std::optional<broker::zeek::Event> detail::to_broker_event(const detail::Event&
     return broker::zeek::Event(ev.HandlerName(), xs, broker_meta);
 }
 
-std::optional<detail::Event> detail::to_zeek_event(const broker::zeek::Event& ev) {
+std::optional<zeek::cluster::Event> detail::to_zeek_event(const broker::zeek::Event& ev) {
```

Comment:
Just for consistency, can you pick either `cluster::Event` or the fully-qualified version? It switches back and forth a couple times in this file.

Reply:
Ah, thanks. I tried to stay with `cluster::Event` here, apparently I failed :-)